components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,259 @@
+
+### Required ServiceNow Plugins
+
+Install following ServiceNow plugins:
+
+- System Import Sets `com.glide.system_import_set`, min version 1.0.0.
+- Configuration Management (CMDB) 1.1.
+- Configuration Management for Scoped Apps (com.snc.cmdb.scoped) 1.0.0.
+
+Locate the **System Import Sets**, **CMDB 1.1**, and **Configuration Management for Scoped App** plugins by navigating to the **System Applications** > **All Available Applications** > **All** section in the ServiceNow application.
+
+{{< figure src="/images/automate/snow_integration_plugins.png" alt="Plugins">}}
+
+### Install the Chef Automate Integration App
+
+The Chef Automate Integration App exposes the REST API endpoint for communication between Chef Automate and the ServiceNow instance. Chef Automate sends HTTPS JSON data feeds to the Chef Automate Integration App in a ServiceNow instance to create and update the _ServiceNow CMDB_ tables, client run, and compliance report tables.
+
+1. Navigate to the ServiceNow store at <https://store.servicenow.com>.
+2. Search for **Chef Automate**.
+3. Select the **Chef Automate Integration App**.
+4. Select **Get** and follow the instructions by specifying your ServiceNow credentials.
+5. Open your **ServiceNow Service Management** application
+6. Select **System Applications** > **All Available Applications** > **All** menu.
+7. Find the application using the filter criteria and search bar.
+   You can search for the application by its `Name` or `ID`. If you cannot find an application, you may have to request it from the ServiceNow Store.
+8. Select **Install**.
+
+### Assign User Roles
+
+The ServiceNow users must have the `x_chef_automate.api` role to set up the ServiceNow instance to receive data from Chef Automate. Changing the **Application Properties** defaults requires the `x_chef_automate.admin` role.
+
+The Chef Automate Integration App has three default roles:
+
+- [Role x_chef_automate.admin]({{< relref "#role_x_chef_automate.admin" >}}))
+- [Role x_chef_automate.user]({{< relref "#role_x_chef_automate.user" >}})
+- [Role x_chef_automate.api]({{< relref "#role_x_chef_automate.api" >}})
+
+You can assign these roles to the existing or new ServiceNow users. These roles are part of the package. If there is a need for further restrictions, you can create the required roles and controls. Also, you can associate a single user with more than one role.
+
+Refer to the following pages on how to create users and assign roles in ServiceNow:
+
+- [Creating Users in ServiceNow](https://docs.servicenow.com/csh?topicname=t_CreateAUser.html)
+- [Assigning Roles in ServiceNow](https://docs.servicenow.com/csh?topicname=t_AssignARoleToAUser.html)
+
+## Configuration
+
+You can configure Chef Automate Integration App from your ServiceNow or Chef Automate instance.
+
+`feed_interval`
+: Valid values: `2` or `8` hours, based on the number of nodes. Increase the `feed_interval` to configure more nodes
```

Comment:
This can be any integer value between 2 and 8

Reply:
: The frequency in hours for refreshing the data feed. The duration between data feed refreshes is proportional to the node count, with more nodes requiring higher settings. Type: Integer. Valid values:  Any integer in the range of `2` to `8`. Default: `2`

Is this correct, @vivekshankar1 ?

Reply:
Its correct and confirmed by @vivekshankar1.